.github/workflows/go.yml

```diff
@@ -78,6 +78,12 @@ jobs:
         if: matrix.os == 'ubuntu-latest' && !github.event.pull_request.head.repo.fork
         run: ko build -B --platform linux/amd64,linux/arm/v7,linux/arm64
 
+      - name: Build OCI image using ko , tag and push it to ghcr
+        if: matrix.os == 'ubuntu-latest' && startsWith(github.ref, 'refs/tags/v')
+        run: |
+          tag=$(echo ${{ github.ref }} | cut -c11-)  # get tag name without tags/refs/ prefix.
+          ko build -B --platform linux/amd64,linux/arm/v7,linux/arm64 --tag-only --tags ${tag}
```

Comment:
What does "--tag-only" do?

Reply:
when running locally i get 

**with tag only we get the expected kubecfg:TAG**
```
➜ ko build -B --push=false --platform linux/amd64 --tags v0.0.0 --tag-only
2023/07/19 10:11:55 Using base cgr.dev/chainguard/static:latest@sha256:6b35c7e7084349b3a71e70219f61ea49b22d663b89b0ea07474e5b44cbc70860 for github.com/kubecfg/kubecfg
2023/07/19 10:11:56 Building github.com/kubecfg/kubecfg for linux/amd64
kubecfg:v0.0.0
```

**without tag only we get  kubecfg:TAG@SHA**
```
➜ ko build -B --push=false --platform linux/amd64 --tags v0.0.0           
2023/07/19 10:12:02 Using base cgr.dev/chainguard/static:latest@sha256:6b35c7e7084349b3a71e70219f61ea49b22d663b89b0ea07474e5b44cbc70860 for github.com/kubecfg/kubecfg
2023/07/19 10:12:03 Building github.com/kubecfg/kubecfg for linux/amd64
kubecfg:v0.0.0@sha256:be001e879aaa9aae17a75323fe7dfbd09cf189353b50b0d93d0d5ed7d434c81b
```

Reply:
actually even `without` `--tag-only` the image in the `ghcr` repository is still tagged properly https://github.com/primeroz/kubecfg/pkgs/container/kubecfg%2Fkubecfg/110775981?tag=v0.0.1-fc-test3

![image](https://github.com/kubecfg/kubecfg/assets/2085772/4974c329-993c-455d-a49e-2800fcf98993)

I will remove it